runtime/Cargo.toml

```diff
@@ -25,6 +25,9 @@ extism-manifest = { workspace = true }
 extism-convert = { workspace = true, features = ["extism-path"] }
 uuid = { version = "1", features = ["v4"] }
 libc = "0.2"
+cap-primitives = ">=2.0"
+cap-fs-ext = ">=2.0"
+wiggle = "22.0.0"
```

Comment:
I think this is probably released along with wasmtime, we should use the same range of supported versions as above - it might also be easier maintenance wise to move this up next to the other wasmtime dependencies.

Reply:
i was able to remove `cap-primives` and `cap-fs-ext` and moved up `wiggle`